lib/common/constants.js

```diff
@@ -157,14 +157,28 @@ function constantsFactory (path) {
         Templates: {
             Directory: process.cwd() + '/data/templates'
         },
+        FinishedTaskStates: [
+            'succeeded',
+            'timeout',
+            'cancelled',
+            'failed'
+        ],
+        FailedTaskStates: [
+            'failed',
+            'timeout',
+            'cancelled'
+        ],
         TaskStates: {
             Running: 'running',
             Succeeded: 'succeeded',
             Failed: 'failed',
             Cancelled: 'cancelled',
             Timeout: 'timeout',
-            Pending: 'pending'
+            Pending: 'pending',
+            Finished: 'finished'
         },
+        DefaultTaskDomain: 'default',
+        DefaultTaskLeaseAdjust: 2000,
```

Comment:
It would be better if these Task constants are wrapped in a hierarchical structure just like other constants:
```javascript
Task: {
    DefaultDomain: 'default',
    DefaultLeaseAdjust: 2000,
    States: [...],
    FailedStates: [...],
    FinishedStates: [...]
}
```